src/app/app/app.component.ts

```diff
@@ -12,8 +14,22 @@ import { LoggerService } from '@ansyn/ansyn';
 export class AppAnsynComponent implements AfterViewInit {
 	mobileOrTable: boolean = this.deviceService.isMobile() || this.deviceService.isTablet();
 	validBrowser: boolean = ['chrome', 'safari'].includes(this.deviceService.browser.toLowerCase()) && !this.mobileOrTable;
+	keysPressed: any = {};
 
-	constructor(public renderer: Renderer2, @Inject(DOCUMENT) protected document: Document, private deviceService: DeviceDetectorService, private loggerService: LoggerService) {
+	constructor(store: Store, public renderer: Renderer2, @Inject(DOCUMENT) protected document: Document, private deviceService: DeviceDetectorService, private loggerService: LoggerService) {
```

Comment:
use event.ctrlKey to check if control is pressed.
add todo to change this when the key listener service is insert.